src/exchange/account/add/account-add.html

```diff
@@ -114,19 +114,10 @@
                            data-ng-if="ctrl.showConfirmPwdError"></small>
                 </div>
 
-                <div class="oui-checkbox">
-                    <input type="checkbox" class="oui-checkbox__input" id="legalWarning"
-                           data-ng-model="ctrl.valid.legalWarning">
-                    <label class="oui-checkbox__label-container" for="legalWarning">
-                        <span class="oui-checkbox__label"
-                              data-ng-bind-html="tr('exchange_ACTION_add_account_legal_warning')"></span>
-                        <span class="oui-checkbox__icon">
-                            <span class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></span>
-                            <span class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></span>
-                            <span class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></span>
-                        </span>
-                    </label>
-                </div>
+                <oui-checkbox id="legalWarning"
+                    model="ctrl.valid.legalWarning"
```

Comment:
`data-` for non HTML5 attributes

Reply:
See my comment here : https://github.com/ovh-ux/ovh-module-emailpro/pull/26#discussion_r159619301